### PR TITLE
Add better filters

### PR DESCRIPTION
**Pull request by amikhalev**

### CODE REVIEW

File: client/tutorSessionFilters.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        switch (filterData.timeRange) {
+            case 'morning':
+                if (!startTime.isBetween(startTime.clone().set({hour: 6}),
+                        startTime.clone().set({hour: 10}))) {
+                    return false;
+                }
+                break;
+            case 'lunch':
+                if (!startTime.isBetween(startTime.clone().set({hour: 11}),
+                        startTime.clone().set({hour: 1}))) {
```
Comment (a-bainbridge) [inline client/tutorSessionFilters.ts]: shouldn't this be `{hour: 13}`?

File: common/sessionFilters.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    let endTime: moment.Moment | undefined;
+    if (typeof query.timeRange === 'string') {
+        query.timeRange = query.timeRange.trim().toLowerCase() as any;
+        switch (query.timeRange) {
+            case 'morning':
+                startTime = moment({ hour: 6 }); endTime = moment({ hour: 11 }); break;
+            case 'lunch':
+                startTime = moment({ hour: 11 }); endTime = moment({ hour: 1 }); break;
+            case 'after_school':
+                startTime = moment({ hour: 1 }); endTime = moment({ hour: 3, minute: 30 }); break;
```
Comment (a-bainbridge) [inline common/sessionFilters.ts]: after_school should be school-specific

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tutor-match",
+  "name": "client",
```
Comment (a-bainbridge) [inline package.json]: ?

File: common/sessionFilters.ts (hunk 2 above)
Comment (amikhalev) [reply common/sessionFilters.ts]: I think this is more of a *could* than a *should*. I purposefully made it a pretty broad range

Comment (amikhalev) [reply common/sessionFilters.ts]: And there's going to be a way to do a custom time range, anyways

**Pull request merged**